src/Application.vala

```diff
@@ -4,14 +4,28 @@
  */
 
 public class Dock.Application : Gtk.Application {
+    private const string LAUNCH_INDEX = "launch-index";
+
+    private const OptionEntry[] OPTIONS = {
+        { LAUNCH_INDEX, 'i', OptionFlags.NONE, OptionArg.INT, null, "Launch the app that's currently at the given index. Should only be used if dock is already running", "INT" },
+    };
```

Comment:
I'm kind of skeptical about this. I'm not sure why someone would want to script this instead of launching the app directly :sweat_smile:

Reply:
Removed it :)